src/main/java/net/william278/velocitab/player/TabPlayer.java

```diff
@@ -98,11 +98,20 @@ public CompletableFuture<Component> getDisplayName(@NotNull Velocitab plugin) {
 
     }
 
+    @NotNull
+    public CompletableFuture<String> getNametag(@NotNull Velocitab plugin) {
+        final String serverGroup = plugin.getSettings().getServerGroup(getServerName());
+        return Placeholder.replace(plugin.getSettings().getNametag(serverGroup), plugin, this)
+                .thenApply(formatted -> plugin.getFormatter().format(formatted)
+                        .replace(player.getUsername(), "%username%"));
+
+    }
+
     @NotNull
     public String getTeamName(@NotNull Velocitab plugin) {
         return plugin.getSettings().getSortingElementList().stream()
                 .map(element -> element.resolve(this, plugin))
-                .collect(Collectors.joining("-"));
+                .collect(Collectors.joining("-")) + player.getUniqueId().toString().substring(0, 3);
     }
```

Comment:
This will probably break if you have long role names and use all three available sorting keys I think, I think,  as the UUID will fall off the end of the team name character limit.

The plan eventually is for the team name thing to just use the same placeholder text system as other parts of the plugin, and to make the "comparable role weight" a placeholder that can be used here, to allow for sorting by PAPI stuff.

Regardless, I'd like this to only be set _if_ a nametag is being used in the group the player is currently in.